android/build.gradle

```diff
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion "23.0.1"
+  compileSdkVersion 25
+  buildToolsVersion "25.0.1"
```

Comment:
@lucasbento The moment react-native target API 25, I'm all for it. But as of today, close to everybody has API 23 (mostly because of the [getting started](https://facebook.github.io/react-native/docs/getting-started.html#3-install-the-android-6-0-marshmallow-sdk))